src/table/table.scss

```diff
@@ -16,75 +15,205 @@
+  &.iui-fixed > .iui-table-body {
+    max-height: $iui-baseline * 27;
     overflow-x: hidden;
     overflow-y: scroll;
     overflow: overlay;
   }
+}
 
-  .iui-select .iui-select-button {
-    background-color: transparent;
-    border: none;
+@mixin iui-table-header {
+  width: 100%;
+  user-select: none;
+  @include themed {
+    background-color: t(iui-color-background-3);
   }
 
-  .iui-tables-row {
-    margin-top: -1px;
+  .iui-row {
     display: flex;
     flex-grow: 1;
     width: 100%;
-    border-top: 1px solid transparent;
-    border-left: 1px solid transparent;
-    border-right: 1px solid transparent;
+  }
+
+  .iui-cell:not(.iui-slot) {
+    @include iui-focus;
+    cursor: pointer;
+    position: relative;
+    overflow: visible;
+
+    > .iui-borderless {
+      margin-left: $iui-xs;
+
+      &:not(.iui-active) {
+        visibility: hidden;
+      }
+    }
+
+    > .iui-resizer {
+      height: 100%;
+      width: $iui-m;
+      position: absolute;
+      top: 0;
+      right: 0;
+      transform: translateX(50%);
+      touch-action: none;
+      cursor: ew-resize;
+      z-index: 100;
+      display: none;
+        
+      > .iui-resizer-bar {
+        height: 100%;
+        width: $iui-xxs;
+        margin: 0 auto;
+        transition: background-color $iui-speed-fast ease-out, width $iui-speed-fast ease-out;
+        @include themed {
+          background-color: t(iui-color-background-5);
+        }
+      }
+
+      &:hover > .iui-resizer-bar {
+        width: $iui-xs;
+        @include themed {
+          background-color: t(iui-color-foreground-primary);
+        }
+      }
+    }
+
+    &:hover,
+    &:focus,
+    &:focus-within {
+      @include themed {
+        background-color: t(iui-color-background-4);
+      }
 
+      .iui-sort,
+      .iui-borderless {
+        visibility: visible;
+      }
+    }
+
+    &:hover {
+      > .iui-resizer {
+        display: block;
+      }
+    }
+  }
+
+  // Sort icon
+  .iui-sort {
+    visibility: hidden;
+    margin-left: auto;
+    width: $iui-icons-default;
+    height: $iui-icons-default;
     @include themed {
-      border-bottom: solid 1px t(iui-color-background-4);
+      fill: t(iui-icons-color-actionable);
     }
+  }
 
-    > .iui-tables-slot > .iui-tables-menu {
-      visibility: hidden;
+  // Sorted column
+  .iui-sorted {
+    @include themed {
+      background-color: t(iui-color-background-4);
+    }
+
+    .iui-sort {
+      visibility: visible;
+        
+      &,
+      > * {
+        @include themed {
+          fill: t(iui-icons-color-actionable);
+        }
+      }
+    }
+  }
+
+  // Double header
+  .iui-row + .iui-row {
+    @include themed {
+      border-top: 1px solid t(iui-color-background-5);
     }
+  }
+}
+
+@mixin iui-table-body {
+  width: 100%;
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+  
+  .iui-row {
+    display: flex;
+    flex-grow: 1;
+    width: 100%;
+    margin-top: -1px;
```

Comment:
Is this for double borders (when active applied) or something else?

Reply:
You are correct.

With `margin-top: -1px;` set:
![Screen Shot 2021-06-01 at 3 00 08 PM](https://user-images.githubusercontent.com/849817/120376589-111a3c80-c2ea-11eb-841f-e9992fa6158b.png)

Without it:
![Screen Shot 2021-06-01 at 3 00 50 PM](https://user-images.githubusercontent.com/849817/120376680-298a5700-c2ea-11eb-87b4-86efc99fbf9e.png)

It's hard to spot, so here is a zoomed in screenshot showing the border line appearing above Folder Name 2.
![Screen Shot 2021-06-01 at 3 05 22 PM](https://user-images.githubusercontent.com/849817/120377192-cbaa3f00-c2ea-11eb-8ef6-ca1a243d3e6d.png)

Reply:
If that solve the issues of not being able to style previous sibling based on next sibling, you might want to consider looking at toggling selection dynamicly to see if content is moving of few pixel when it's selected and when it's not. Styles of selection was taking in consideration that when it's selected, the padding/border was change so the content doesn't budge.